lib/schema.ts

```diff
@@ -794,6 +796,9 @@ const LiteLLMConfig = z.object({
  * @property {string} s3BucketModels - S3 bucket for models.
  * @property {string} mountS3DebUrl - URL for S3-mounted Debian package.
  * @property {string[]} [accountNumbersEcr=null] - List of AWS account numbers for ECR repositories.
+ * @property {boolean} [deployRag=false] - Whether to deploy RAG stacks.
+ * @property {boolean} [deployChat=false] - Whether to deploy chat stacks.
+ * @property {boolean} [deployUi=false] - Whether to deploy UI stacks.
```

Comment:
minor nitpick on these since they're defaulting to `true` now. thanks for adjusting those- that will help reduce a lot of confusion from someone accidentally deleting the UI :)